### PR TITLE
add chartbeat to plugins

### PR DESCRIPTION
**Pull request by kimlarocca**
https://nypublicradio-digital.atlassian.net/browse/GOTH-483

### CODE REVIEW

File: plugins/chartbeat.client.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export default defineNuxtPlugin(() => {
+  /* eslint-disable */
+  (function() {
+    /** CONFIGURATION START **/
+    const _sf_async_config = window._sf_async_config = (window._sf_async_config || {});
+    _sf_async_config.uid = 16246; //CHANGE THIS TO YOUR ACCOUNT ID
```
Comment (bonomite) [inline plugins/chartbeat.client.js:6]: should this ID be an env var?

**Review by bonomite — COMMENTED** [review]
one tiny comment

File: plugins/chartbeat.client.js (hunk 1 above)
Comment (kimlarocca) [reply plugins/chartbeat.client.js:6]: It doesn't change for demo or prod - but this is how it was setup on the old site (I didn't set it up)

Comment (bonomite) [reply plugins/chartbeat.client.js:6]: security?

**Review by bonomite — APPROVED** [review]
cool

**Pull request merged**